pkg/compliance/checks/docker_check.go

```diff
@@ -123,7 +123,9 @@ func newDockerInfoInstance(ctx context.Context, client env.DockerClient) (eval.I
 	}
 
 	return eval.NewInstance(
-		nil,
+		eval.VarMap{
+			compliance.DockerContainerInspect: info,
```

Comment:
I believe this should be `DockerInfoInspect`
```suggestion
			compliance.DockerInfoInspect: info,
```

Reply:
It should: autocomplete got me here. But now I'm confused why it works?

Reply:
ah, I fixed it locally but forgot to commit the change

Reply:
It works because the rego evaluator only look at the second part (after the dot). I don't think the old evaluator would work on the other end